### ISSUE TITLE
Node positions from click

### ISSUE THREAD

**Issue by ekkis** [Linkurious/linkurious.js#294]
just a small question: the users double-click on the canvas I get x, y coordinates, which I use to position a new node on the graph, however, the nodes never appear where I click... as if the two coordinate systems were offset.  is there some translation that needs to happen when setting the node's x and y?

**Comment by sheymann** [comment]
Yes you need to convert the coordinates of the point from the screen to the graph as follows:

```js
var graphCoords = sigmaInstance.camera.cameraPosition(screenX ,screenY);

sigmaInstance.graph.addNode({
  //...
  x: graphCoords.x,
  y: graphCoords.y
})
```

Notice that the point (0,0) is the center of the screen.

**Comment by ekkis** [comment]
that seems to put the node even farther away.  the coordinates I have I'm getting from the doubleClickStage event's data.captor clientX and clientY.  should I be using a different source?

**Comment by sheymann** [comment]
How about using `event.data.captor.x` and `event.data.captor.y` from the sigma `dblclick` event?
